.pre-commit-config.yaml

```diff
@@ -56,3 +56,21 @@ repos:
         language: golang
         require_serial: true
         files: ^kubectl-plugin/
+
+  - repo: local
+    hooks:
+      - id: check-kubeconform-version
+        name: kubeconform version check
+        entry: bash -c 'version="0.6.7"; [ "$(kubeconform -v | grep -oP "(?<=v)[\d\.]+")" = "$version" ] || { echo "kubeconform version is not $version"; }'
```

Comment:
Can you replace this line with `bash -c 'version="0.6.7"; [ "$(kubeconform -v | grep -oP "(?<=v)[\d\.]+")" = "$version" ] || { echo "kubeconform version is not $version"; exit 1; }'`

Also I find that I made a mistake in line 34, can you replace it with `bash -c 'version="1.60.3"; [ "$(golangci-lint --version | grep -oP "(?<=version )[\d\.]+")" = "$version" ] || { echo "golangci-lint version is not $version"; exit 1; }'`? Thanks.

Otherwise LGTM.